frontend/lib/parsing/ParserContextImpl.h

```diff
@@ -2533,44 +2533,41 @@ ParserContext::buildAggregateTypeDecl(YYLTYPE location,
 
   auto contentsList = consumeList(contents);
 
-  owned<AstNode> inheritExpr;
+  AstList inheritExprs;
   if (optInherit != nullptr) {
     if (optInherit->size() > 0) {
-      if (parts.tag == asttags::Record) {
-        CHPL_PARSER_REPORT(this, RecordInheritanceNotSupported, inheritLoc,
-                           parts.name.str());
-      } else if (parts.tag == asttags::Union) {
+      if (parts.tag == asttags::Union) {
```

Comment:
We need to support unions implementing an interface eventually, but that's not important right now. So, could you add a TODO comment here?